docs/v1/src/pages/docs/react/Suspense.en.mdx

```diff
@@ -27,33 +27,33 @@ Control multiple `<Suspense/>`s default fallback with `<SuspensiveProvider/>`. D
 
 </Callout>
 
-### Avoid Server side rendering (CSROnly)
+### Avoid Server side rendering (csrOnly)
```

Comment:
I think `CSROnly` is fine as it is, because `CSR` is a shortened version (Client Side Rendering), so it's not awkward to capitalize it all. 

If it's awkward to capitalize a prop name, why not `onlyCSR` or `clientOnly`?

Reply:
Oh, clientOnly prop is better than csrOnly! Cool! I will change it soon